ui/src/components/LoginForm/SessionTable.jsx

```diff
@@ -5,6 +5,35 @@ import { LuExternalLink } from 'react-icons/lu';
 import SessionDateTime from './SessionDateTime';
 import styles from './SessionTable.module.css';
 
+function isDateInRange(actualStartDate, actualEndDate, rangeInDays) {
```

Comment:
Until this needs to be generic, I wouldn't bother making it so (as it might not ever need to be generic):

```js
function isAroundNow(session) {
  const startDate = parseDate(session.actual_start_date);
  const endDate = parseDate(session.actual_end_date);

  ...
}
```

Could you maybe also move this function into a `util.js` file in the same folder?

Arguably, `SelectProposal.jsx`, `SessionTable.jsx`, etc. and this `util.js` file should maybe be moved into a `SessionProposal` folder. They're not really part of the `LoginForm`.

Reply:
> Arguably, SelectProposal.jsx, SessionTable.jsx, etc. and this util.js file should maybe be moved into a SessionProposal folder. They're not really part of the LoginForm.

Completely agree. But it is probably to be addressed in a separated issue as it is beyond the current issue.
https://github.com/mxcube/mxcubeweb/issues/1568